### DISCUSSION TITLE
Improve handling of packed labels in Loki Datasource

### DISCUSSION THREAD

**Discussion by itsthatriver** [grafana/grafana#38947] [Data Sources]
The upcoming 2.4 release of Loki makes it much easier to improve Loki performance with label packing by adding support for Out Of Order logs. The problem is that if you now look at metrics on Cortex for `{pod="PODNAME"}` and in Loki the pod label is packed, if you try to jump directly from metrics to logs the auto-populated query won't work.

Similar to how in the Tempo data source you can call out specific tags on a Logs data source to aid in navigating from logs to traces, it would be great if you could provide a list of labels on the Loki datasource that were known to be packed so that when you navigate from metrics to logs you could go from `{app="api",pod="PODNAME"}` in Cortex to `{app="api"} | unpack | pod="PODNAME"` or something to that effect.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
